## strict-gcc43-include.diff by Jens Peter Secher <jps@debian.org>
##
## Explicitly include cstring to allow compilation with GCC 4.3.

Index: libcrypto++/misc.h
===================================================================
--- libcrypto++.orig/misc.h	2010-08-09 20:42:43.000000000 +0200
+++ libcrypto++/misc.h	2010-08-09 20:42:47.000000000 +0200
@@ -39,6 +39,7 @@
 #if defined(__GNUC__) && defined(__linux__)
 #define CRYPTOPP_BYTESWAP_AVAILABLE
 #include <byteswap.h>
+#include <cstring>
 #endif
 
 NAMESPACE_BEGIN(CryptoPP)
